src/test/java/seedu/duke/logic/commands/RouteMapCommandTest.java

```diff
@@ -0,0 +1,36 @@
+package seedu.duke.logic.commands;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.Bus;
+import seedu.duke.BusData;
+import seedu.duke.BusStops;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RouteMapCommandTest {
+    private static ArrayList<Bus> busesInZoneA = new ArrayList<>();
+    private static final BusStops[] AA1 = {
+        BusStops.PGP,
+        BusStops.KENTRIDGEMRTSTATION,
+        BusStops.OPPUNIVERSITYHEALTHCENTRE,
+        BusStops.YUSOFISHAKHOUSE,
+        BusStops.CENTRALLIBRARY,
+        BusStops.KENTRIDGE,
+        BusStops.MUSEUM,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.UNIVERSITYHEALTHCENTRE,
+        BusStops.OPPKENTRIDGEMRTSTATION,
+        BusStops.PGPR
+    };
+
+    @Test
+    void executeCommand_success() {
+        busesInZoneA.add(new Bus("AA1", AA1));
+        BusData.createBusList(busesInZoneA);
```

Comment:
Since you have just one test, using createBusList() here is ok. If you have more than one tests for BusData, remember that calling it again might cause duplicate entries in the list.
A possible way to do such initialization is using a @BeforeAll method.

Reply:
This is something I found from my research. Do let me know if there is a better alternative!